Description: Strike references to removed non-DFSG sources from build files
Forwarded: not-needed
From: Felix Lechner <felix.lechner@lease-up.com>
Index: wolfssl-5.6.6/Makefile.am
===================================================================
--- wolfssl-5.6.6.orig/Makefile.am
+++ wolfssl-5.6.6/Makefile.am
@@ -182,25 +182,6 @@ include linuxkm/include.am
 include zephyr/include.am
 include RTOS/nuttx/include.am
 
-# Exclude references to non-DFSG sources from build files
-if !BUILD_DISTRO
-include mqx/util_lib/Sources/include.am
-include mqx/wolfcrypt_benchmark/Sources/include.am
-include mqx/wolfcrypt_test/Sources/include.am
-include mqx/wolfssl/include.am
-include mqx/wolfssl_client/Sources/include.am
-include mplabx/include.am
-include mplabx/wolfcrypt_benchmark.X/nbproject/include.am
-include mplabx/wolfcrypt_test.X/nbproject/include.am
-include mplabx/wolfssl.X/nbproject/include.am
-include mcapi/include.am
-include mcapi/wolfcrypt_mcapi.X/nbproject/include.am
-include mcapi/wolfcrypt_test.X/nbproject/include.am
-include mcapi/wolfssl.X/nbproject/include.am
-include mcapi/zlib.X/nbproject/include.am
-include tirtos/include.am
-include IDE/include.am
-endif
 include scripts/include.am
 
 if BUILD_LINUXKM
